aggregation-worker/pom.xml

```diff
@@ -132,34 +125,20 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- Deploy docker artifact to Amazon ECS -->
             <plugin>
-                <groupId>com.spotify</groupId>
-                <artifactId>dockerfile-maven-plugin</artifactId>
-                <version>1.3.5</version>
-                <configuration>
-                    <repository>${aws.docker.repository.uri}</repository>
-                    <googleContainerRegistryEnabled>false</googleContainerRegistryEnabled>
-                    <verbose>true</verbose>
-                    <tag>latest</tag>
-                </configuration>
+                <artifactId>exec-maven-plugin</artifactId>
+                <groupId>org.codehaus.mojo</groupId>
                 <executions>
                     <execution>
-                        <id>package</id>
-                        <goals>
-                            <goal>build</goal>
-                        </goals>
-                    </execution>
-                    <execution>
-                        <id>install</id>
+                        <id>docker-deploy</id>
+                        <phase>install</phase>
```

Comment:
Would deploy be better?  I use install a lot to install artefacts into my local repository - I wouldn't expect it to push the image up to the registry - deploy is used to push to publishing repositories.